local_planner/test/test_planner_functions.cpp

```diff
@@ -447,6 +447,7 @@ TEST(PlannerFunctions, getCostMatrixNoObstacles) {
   Eigen::Vector3f position(0.f, 0.f, 0.f);
   Eigen::Vector3f goal(0.f, 5.f, 0.f);
   Eigen::Vector3f last_sent_waypoint(0.f, 1.f, 0.f);
+  float heading = 0.f;
```

Comment:
Should another test be added to cover heading changes in the cost function?

Reply:
yes that might be a good idea